src/style.rs

```diff
@@ -365,20 +365,20 @@ impl Command for SetAttr {
     }
 }
 
-/// A command to print the styled object.
+/// A command to print the styled content.
 ///
-/// See [`StyledObject`](struct.StyledObject.html) for more info.
+/// See [`StyledContent`](struct.StyledContent.html) for more info.
 ///
 /// # Notes
 ///
 /// Commands must be executed/queued for execution otherwise they do nothing.
-pub struct PrintStyledFont<D: Display + Clone>(pub StyledObject<D>);
+pub struct PrintStyledFont<D: Display + Clone>(pub StyledContent<D>);
```

Comment:
I don't like `PrintStyledFont` name. @TimonPost can we include it in the #286 and rename it? I doesn't print `Font`,  it prints `StyledContent`.

Not related to this PR, just noticed it.

Reply:
Yea, good idea. Let's move it in that list.